tox.ini

```
@@ -36,8 +46,6 @@ commands = python3 -m flake8 {[common]pyfiles}
 deps = pylint
 commands = python3 -m pylint {[common]pyfiles}
 
-[testenv:shellcheck]
-# shellcheck from Xenial is too old and doesn't support the --severity
-# argument. Let's pull the latest version using shellcheck-py.
+[testenv:shellcheck-tip]
```

For consistency with the tip environments in cloud-init and curtin:

```suggestion
[testenv:tip-shellcheck]
```

---

`pylint-tip` and `flake8-tip` are to be changed too. The intention *was* to have the same envs that we have in curtin and cloud-init, but I got them wrong.